src/client/test/unit/PacWrapper.test.ts

```diff
@@ -9,6 +9,7 @@ import { ITelemetry } from "../../telemetry/ITelemetry";
 class MockContext implements IPacWrapperContext {
     public get globalStorageLocalPath(): string { return ""; }
     public get telemetry(): ITelemetry { return NoopTelemetryInstance; }
+    public get automationAgent(): string { return "PacVscodeTests/0.1.0-dev"; }
```

Comment:
use a product name that's related to this repo's product. i.e. `powerplatform-vscode-tests`

Reply:
updated